app/src/main/java/com/rockthevote/grommet/ui/registration/RegistrationState.kt

```diff
@@ -0,0 +1,14 @@
+package com.rockthevote.grommet.ui.registration
+
+import androidx.annotation.StringRes
+
+sealed class RegistrationState {
+    object Init: RegistrationState()
+    object InProgress : RegistrationState()
```

Comment:
might need an abandoned state, not sure yet though, depends on how "complete" is going to be used/tracked

Reply:
Agreed, this would be a good place for that.